diff --git a/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js b/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js
index 7912e9a..7eba8d3 100644
--- a/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js
+++ b/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js
@@ -122,6 +122,12 @@ async function runServer(_argv, cliConfig, args) {
   _devMiddleware.createDevMiddleware)({
     serverBaseUrl: devServerUrl,
     logger: (0, _createDevMiddlewareLogger.default)(terminalReporter),
+    unstable_experiments:
+      process.env.REACT_NATIVE_NO_FUSEBOX_PREPARE === "1"
+        ? {
+            enableStandaloneFuseboxShell: false,
+          }
+        : {},
   });
   metroConfig.reporter = {
     update(event) {
